Servers/UI/OJS.Servers.Ui/ClientApp/src/components/profile/profile-submissions/ProfileSubmisssions.tsx

```diff
@@ -19,31 +19,28 @@ const ProfileSubmissions = ({ userIsProfileOwner, isChosenInToggle }: IProfileSu
     const { internalUser, isLoggedIn } = useAppSelector((reduxState) => reduxState.authorization);
     const { profile } = useAppSelector((state) => state.users);
 
-    const dispatch = useAppDispatch();
-
-    const [ getUserSubmissionsQuery, {
-        data: userSubmissions,
-        isLoading: areSubmissionsLoading,
-        error: userSubmissionsQueryError,
-    } ] = useLazyGetUserSubmissionsQuery();
-
-    useEffect(() => {
+    const canFetchSubmissions = React.useMemo(() => {
```

Comment:
import `useMemo`. Also there is no need to import `React` at all.

Reply:
Done.